### PR TITLE
Update metrics to account for missed false positives

### PR DESCRIPTION
**Pull request by RitwikGupta**

### CODE REVIEW

File: reference/metric.py [hunk 1]
```diff
@@ -355,9 +355,9 @@ def score(pred, gt, shore_root, distance_tolerance=200, shore_tolerance=2):
                 scene_id,
                 shore_tolerance + distance_tolerance / 1000,
             )
-            print(
-                f"{len(gt_sc_shore)} ground truth, {len(pred_sc_shore)} predictions close to shore"
-            )
+            #print(
+            #    f"{len(gt_sc_shore)} ground truth, {len(pred_sc_shore)} predictions close to shore"
+            #)
```
Comment (jdunnmon) [inline reference/metric.py:360]: Any particular reason to remove this?

Comment (RitwikGupta) [reply reference/metric.py:360]: The constant unbuffered prints to stdout really slow down the scoring. It also messes with the `tqdm` formatting.

Comment (jdunnmon) [reply reference/metric.py:360]: Approving this after manual testing with a set of dummy files.  Repro'd issue, confirm that fix works.

**Review by jdunnmon — APPROVED** [review]

**Comment by jdunnmon** [comment]
Approving this after manual testing with a set of dummy files.  Repro'd issue, confirm that fix works. 

Noting also that this PR updates the analogous `elif` statement in `compute_vessel_class_performance` to be the same as this proposed `elif` statement in `compute_fishing_class_performance`.  This change should not affect output, but is more explicit about the condition.

File: reference/metric.py (hunk 1 above)
Comment (jdunnmon) [reply reference/metric.py:360]: Makes sense, agree, let's leave it commented in case anyone wants to use it.

**Review by timatcambrio — APPROVED** [review]
Tested with a small synthetic ground truth file and a corresponding set of predictions. The update returns the correct number of `tp/fp/tn/fn` for `vessel_class` and `fishing_class`.

**Pull request merged**